powershell/tests/smoketests/Failure.Tests.ps1

```diff
@@ -0,0 +1,62 @@
+Describe "TestBlock1" {
```

Comment:
It would be ideal to use concise, but descriptive names for each **Describe** block.